spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/mongo/MongoProperties.java

```diff
@@ -51,6 +51,11 @@ public class MongoProperties {
 	 */
 	public static final String DEFAULT_URI = "mongodb://localhost/test";
 
+	/**
+	 * Protocol to be used for the MongoDB connection. Ignored if 'uri' is set.
```

Comment:
I was just looking at this and laughed that it matches the polish I've made in 44384. What a coincidence! Nope, you forced push to adapt... You really didn't have to do it, thanks!